src/rpc/core_rpc_server.cpp

```diff
@@ -164,8 +167,27 @@ namespace cryptonote
         ssl_options = epee::net_utils::ssl_options_t{std::move(ssl_allowed_fingerprints), std::move(ssl_ca_path)};
     }
 
+    epee::wipeable_string private_key_passphrase;
+    std::string passphrase_file = command_line::get_arg(vm, arg_rpc_ssl_private_key_passphrase_file);
+    if (!passphrase_file.empty())
+    {
+      std::string passphrase;
+      if (!epee::file_io_utils::load_file_to_string(passphrase_file, passphrase))
+      {
+        MERROR("Failed to load passphrase");
+        return false;
+      }
+
+      // Remove line breaks the user might have inserted
+      boost::trim_right_if(passphrase, boost::is_any_of("\r\n"));
+      private_key_passphrase = passphrase;
```

Comment:
A move constructor exists, `private_key_passphrase = std::move(passphrase);`. The `memwipe` can still be done, but the string should be empty by then.